route/route.go

```diff
@@ -915,6 +962,9 @@ func (r *Router) AddOTLPMuxxer(muxxer *mux.Router) {
 	otlpMuxxer := muxxer.PathPrefix("/v1/").Methods("POST").Subrouter()
 
 	// handle OTLP trace requests
-	otlpMuxxer.HandleFunc("/traces", r.postOTLP).Name("otlp")
-	otlpMuxxer.HandleFunc("/traces/", r.postOTLP).Name("otlp")
+	otlpMuxxer.HandleFunc("/traces", r.postOTLPTrace).Name("otlp")
```

Comment:
We don't use `.Name` for its intended purpose -- it's basically just a comment for us -- but I think it would be worth using different values for the logs and traces muxxer names; they might show up in a log or debugging session sometime.